lib/rubocop/cop/layout/space_around_keyword.rb

```diff
@@ -30,10 +30,13 @@ class SpaceAroundKeyword < Cop
 
         DO = 'do'
         SAFE_NAVIGATION = '&.'
+        NAMESPACE_OPERATOR = '::'
         ACCEPT_LEFT_PAREN =
           %w[break defined? next not rescue return super yield].freeze
         ACCEPT_LEFT_SQUARE_BRACKET =
           %w[super yield].freeze
+        ACCEPT_NAMESPACE_OP =
+          %w[super].freeze
```

Comment:
I think it's enough to use string object instead of array object.

Reply:
Change to string and matching using `==` instead of `include?`